src/style.css

```diff
@@ -0,0 +1,18 @@
+.page-stile {
+  font: 13px arial, sans-serif;
+}
+
+.title {
+  font-size: 17px;
+  margin: 0;
+}
+
+.articles {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+.href {
+  color: green;
+}
```

Comment:
The name of the element class should make a sense. It should describe what the role of the element is on this page or describe its state. href is attribute, that make no sense, when you use it like classname